src/service/poc_mobile.proto

```diff
@@ -438,6 +438,90 @@ message radio_reward {
   repeated boosted_hex boosted_hexes = 10;
 }
 
+enum boosted_hex_status {
```

Comment:
if you move this enum into the radio_reward_v2, you could drop the `boosted_hex_status_` prefix on each element ..make things less verbose.

Reply:
From my testing, `boosted_hex_status` cannot be moved into `radio_reward_v2` because it shares the same name as the field. It appears the `boosted_hex_status_` prefix can be dropped from the enum values anyways. Seeing as this is going out soon, and renaming fields is not a problem with proto enums as long as we don't change the order, I'm going to leave that for a less pressing time.